src/transformers/models/llava_next_video/diff_llava_next_video.py

```diff
@@ -0,0 +1,27 @@
+from transformers import LlavaNextConfig
```

Comment:
```suggestion
from transformers.models.llava.config_llava import LlavaNextConfig
```

Reply:
Oke, that will fix the "diff_my_model" for overwriting args, right? I wlll check tomorrow :)

Reply:
@ArthurZucker thanks, I changed it to the actual path of LLava-next and now the generated config has no imports at all 🥲  I will leave it as it was, I don't see any difference between two except that the second does no imports

`from transformers.models.llava_next.configuration_llava_next import LlavaNextConfig`